eng/restore/runtime/runtime.depproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(RuntimeOS)' == 'webassembly'">$(ToolRuntimeRID)</RuntimeIdentifier>
```

Comment:
This deserves a comment.  We're using ToolRuntime as a placeholder for the real corelib/runtime components until we have an actual set of runtime bits to consume for webassembly.

Reply:
Makes sense.

Reply:
Makes sense.